src/More/Licenses.tsx

```diff
@@ -22,7 +23,7 @@ const Licenses: FunctionComponent = () => {
   const infoEmailLink = "mailto:info@pathcheck.org"
   const pathCheckWebAddress = "pathcheck.org"
   const pathCheckUrl = "https://pathcheck.org/"
-  const privacyPolicyUrl = "https://pathcheck.org/privacy-policy/"
+  const privacyPolicyUrl = env.PRIVACY_POLICY_URL
```

Comment:
Should we have a fallback value for it? Or perhaps not show the link if no privacy policy URL is present? It sucks to do but the data allows for it to happen 😨

Reply:
Yeah good call.